scripts/winesapos-install.sh

```diff
@@ -227,6 +229,19 @@ if [[ "${WINESAPOS_BUILD_CHROOT_ONLY}" == "false" ]]; then
     done
 
     echo "Mounting partitions complete."
+    # Mohamed: Once formatting is completed, you can now use ./pifs to convert the filesystem to πfs
+    if [[ "${WINESAPOS_PIFS}" == "true" ]]; then
+        CWD = $PWD
+        cd /tmp
+        git clone https://github.com/aya/pifs
+        cd pifs
+        ./autogen.sh
+        ./configure
+        make
```

Comment:
After a `make`, you can run `sudo make install` to install the binary to: `/usr/local/bin/pifs`